### PR TITLE
chore (refs DPLAN-11413): add DpContextualHelp instead of v-tool…

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://demoseurope.youtrack.cloud/issue/DPLAN-11413

- add DpContextualHelp instead of v-tooltip directive.
- add it for contextual_help.html.twig & label.html.twig

### CODE REVIEW

**Review by spiess-demos — COMMENTED** [review]
Some ideas that do not directly relate to changed code:
- all occurrences of `o-tooltip--question-circle` in usages of `contextualHelp('` or `contextual_help.html.twig` could be removed as redundant, if we'd add `fa fa-question-circle` to the noscript block of `contextual_help.html.twig`.
-

File: templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig [hunk 1]
```diff
@@ -53,9 +53,8 @@
     {% endif %}
 </label>
 {% if tooltip|default %}
-    <i
-        class="{{ 'fa fa-question-circle absolute mt-0.5 top-0 right-0'|prefixClass(omitPrefix) }}"
-        :aria-label="Translator.trans('contextual.help')"
-        v-tooltip="'{{ tooltip|trans }}'">
-    </i>
+    <dp-contextual-help
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig]: Done in https://github.com/demos-europe/demosplan-core/pull/2970/commits/809e2981324cb8c11ac6b0acd8b37dae491ecf28

**Comment by ahmad-demos** [comment]
> All occurrences of `o-tooltip--question-circle` in usages of `contextualHelp('` or `contextual_help.html.twig` could be removed as redundant, if we'd add `fa fa-question-circle` to the noscript block of `contextual_help.html.twig`.

Done in:
https://github.com/demos-europe/demosplan-core/pull/2970/commits/a8a30978160ccb5976692ebce902973a06b499a0
https://github.com/demos-europe/demosplan-core/pull/2970/commits/59cfe1e6e0c5c539559d1beb13e13baff3c4cbde
https://github.com/demos-europe/demosplan-core/pull/2970/commits/2537789bf44613d9bd062ca7d5ed828298e884f4
https://github.com/demos-europe/demosplan-project-diplanrog/pull/109
https://github.com/demos-europe/demosplan-project-diplanbau/pull/207
https://github.com/demos-europe/demosplan-project-diplanfest/pull/83
https://github.com/demos-europe/demosplan-project-teilhabe/pull/51

File: templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig [hunk 2]
```diff
@@ -53,9 +53,8 @@
...
 </label>
 {% if tooltip|default %}
-    <i
-        class="{{ 'fa fa-question-circle absolute mt-0.5 top-0 right-0'|prefixClass(omitPrefix) }}"
-        :aria-label="Translator.trans('contextual.help')"
-        v-tooltip="'{{ tooltip|trans }}'">
-    </i>
+    {% include '@DemosPlanCore/Extension/contextual_help.html.twig' with {
+        helpText: tooltip|trans,
+        cssClasses:'mt-0.5 top-0 right-0'
```
Comment (spiess-demos) [inline templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig:58]:
```suggestion
        cssClasses:'absolute mt-0.5 top-0 right-0'
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]
just one small concern left.

File: templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig (hunk 2 above)
Comment (ahmad-demos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig:58]: Screenshot 2024-04-22 at 10-12-50 Stellungnahme Details Test_2 9 2_I Bauleitplanung Online
When use absolute!

Comment (salisdemos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig:58]: then You have to add `relative` to the parent element

Comment (spiess-demos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig:58]: I think it would be better to move the icon within the label element.

Comment (ahmad-demos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig]: Done in https://github.com/demos-europe/demosplan-core/pull/2970/commits/e3d1d2af0b36e3750c5a3debb63dc21aaf8e741d

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**